armsrc/Standalone/hf_15693sniff.c

```diff
@@ -0,0 +1,128 @@
+//-----------------------------------------------------------------------------
```

Comment:
Please use the same copyright and license header as in the other Standalone modes as it has been modified a couple of days ago, so 
```
//-----------------------------------------------------------------------------
// Copyright (C) 2021 Nathan Glaser <startrk1995@gmail.com>
// Copyright (C) Proxmark3 contributors. See AUTHORS.md for details.
//
// This program is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// See LICENSE.txt for the text of the license.
//-----------------------------------------------------------------------------
// main code for standalone HF/iso15693 Sniff to flash
//-----------------------------------------------------------------------------
```

Reply:
Done